spec/models/instance/settings_spec.rb

```diff
@@ -1,45 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe Instance::Settings, type: :model do
-  ALL_COMPONENTS = [:A, :B, :C, :D, :E, :F]
-  ENABLED_COMPONENTS = [:A, :C, :E]
+  ALL_COMPONENTS = [:A, :B, :C, :D, :E, :F].freeze
```

Comment:
You can try to do `puts ALL_COMPONENTS ` in other specs files, you will get the same values.